packages/core/src/login/oidc/IClientRegistrar.ts

```diff
@@ -111,26 +120,26 @@ export async function handleRegistration(
   // or it is not compliant but the client_id isn't an IRI (we assume it has already
   // been registered with the IdP), then the client registration information needs
   // to be stored so that it can be retrieved later after redirect.
-  await storageUtility.setForUser(options.sessionId, {
+  const clientDetails: IClientDetails = {
```

Comment:
We should probably revisit this, because I don't think it necessarily makes sense given this is only for solid-oidc or static clients both of which don't expire, as far as I'm aware. We also don't use the stored `clientName` value anywhere later, iirc.